packages/calling/src/CallingClient/CallingClient.ts

```diff
@@ -142,7 +142,7 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
         retry = true;
       }
 
-      if (retry && this.webex.internal.mercury.connected) {
+      if (retry && this.webex.internal.mercury.connected && line.mobiusDeviceId) {
```

Comment:
Is there a possibility to add a UT for this by simulating this scenario? Also there maybe a possibility to test this manually also, have you tried to test it?

Reply:
ack: updated